src/components/ApplicationDownloadLink.js

```diff
@@ -156,64 +155,21 @@ export default function ApplicationDownloadLink(props) {
   const internalCollaborators = getCollaborators(darInfo, 'internalCollaborators');
   const externalCollaborators = getCollaborators(darInfo, 'externalCollaborators');
   const labCollaborators = getCollaborators(darInfo, 'labCollaborators');
-  const researcherProps = getPropertyValuesFromUser(researcherProfile);
-  const location = (researcherProps.city).concat(', ').concat(researcherProps.state);
+
   // Use PDFViewer during development to see changes to the document immediately
   // return h(PDFViewer, {width: 1800, height: 800}, [
-
   const document = h(Document, {}, [
     h(Page, {style: styles.page}, [ //Researcher Info Page
       h(View, {}, [
         h(Text, {style: styles.header}, [`${darInfo.darCode} Application`]),
         h(Text, {style: styles.subHeader}, ['Applicant Information']),
-        h(View, {style: styles.flexboxContainer}, [
-          h(SmallLabelTextComponent, {
-            label: 'NIH eRA Commons ID',
-            text: `${researcherProps.eraCommonsId}`,
-            style: {marginRight: 30}
-          }),
```

Comment:
This block is something we still need. The era commons id is actually a prop on the user now - this is relatively old code.